webpack/sass/main.scss

```diff
@@ -1,4 +1,18 @@
+
 @import "../node_modules/@creativecommons/vocabulary/scss/vocabulary.scss";
+@import "./home-page.scss";
+@import "./blog-page.scss";
+@import "./single-post-page.scss";
+@import "./author-post-page.scss";
+@import "./series-page.scss";
+@import "./categories-page.scss";
+@import "./authors-page.scss";
+@import "./guidelines.scss";
+@import "./project-list.scss";
+@import "./project-ideas.scss";
+@import "./search-guideline-page.scss";
```

Comment:
Please update it to the right name it is `./search-guidelines-page.scss`. When you run the project check if it throws any error like this -
![image](https://user-images.githubusercontent.com/34674462/93198670-f9565480-f76a-11ea-9a6a-af3762ec1cb1.png)

Reply:
Push commits after you make sure there are no errors like this. And if you run into something unfamiliar we will be happy to help.